code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -128,6 +128,10 @@ def __init__(self, **kwargs) -> None:
             "AZURE_FORM_RECOGNIZER_KEY"
         )
         # Azure App Insights
+        self.APPINSIGHTS_ENABLED = (
+            os.getenv("APPINSIGHTS_ENABLED", "True").lower() == "true"
+        )
+
         self.APPINSIGHTS_CONNECTION_STRING = os.getenv(
```

Comment:
There seems to be more than 1 AppInsights connection string if you have seen - do you think you can tidy that whilst here? If it is too big then feel free to create a new issue

Reply:
I can't seem to find it. I only see `APPINSIGHTS_CONNECTION_STRING`